### PR TITLE
feat: add authors names

### LINKED ISSUE #3: add authors names

### PR DESCRIPTION
**Pull request by mariembencheikh**
add authors names on books
fix #3

### CODE REVIEW

File: books.txt [hunk 1]
```diff
@@ -1,2 +1,2 @@
-First book
-Second book 
+First book author1
```
Comment (spaenleh) [inline books.txt]: I think that we should add a comma to distinguish the name of the book from the author, what do you think ?

File: books.txt [hunk 2]
```diff
@@ -1,2 +1,2 @@
...
+Second book author2
```
Comment (spaenleh) [inline books.txt]: Same here

**Review by spaenleh — CHANGES_REQUESTED** [review]
Nice work, but I am thinking we should maybe make the format more clear with a comma ?

File: books.txt (hunk 1 above)
Comment (mariembencheikh) [reply books.txt]: @spaenleh  more details please

Comment (spaenleh) [reply books.txt]: I was thinking we could have the format as follows:
```suggestion
First book, author1
```

**Review by spaenleh — APPROVED** [review]

**Pull request merged**